src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_b_fs_yx_fsv16_1x1.cpp

```diff
@@ -163,12 +180,21 @@ bool ConvolutionKernel_b_fs_yx_fsv16_1x1::Validate(const Params& p) const {
     const auto& input = params.inputs[0];
     const auto& output = params.outputs[0];
 
-    const bool bOutputSizes = output.X().v != input.X().v || output.Y().v != input.Y().v || output.Feature().v % 16 != 0;
+    GPU_DEBUG_INFO << "input: " << input.Batch().v << ", " << input.Feature().v << ", " << input.Y().v << ", " << input.X().v << std::endl;
+    GPU_DEBUG_INFO << "output: " << output.Batch().v << ", " << output.Feature().v << ", " << output.Y().v << ", " << output.X().v << std::endl;
+
+    const bool bOutputSizes = (!params.is_shape_agnostic && (output.X().v != input.X().v || output.Y().v != input.Y().v)) ||
+                                output.Feature().v % 16 != 0;
```

Comment:
```suggestion
    const bool bOutputSizes = !params.has_dynamic_tensors() && ((output.X().v != input.X().v || output.Y().v != input.Y().v) ||
                                output.Feature().v % 16 != 0);
```
Please check if it's your intention.

Reply:
Applied, I check whether dynamic in each dim.